src/action/base/create_or_merge_nix_config.rs

```diff
@@ -413,6 +413,8 @@ impl Action for CreateOrMergeNixConfig {
             &[
                 "# Generated by https://github.com/DeterminateSystems/nix-installer.",
                 "# See `/nix/nix-installer --version` for the version details.",
```

Comment:
IMHO it would be better to do `new_config.push_str("# Generated by.....\n"); new_config.push_str("# See ........\n");` rather than just splatting in some more newlines.

Reply:
Good idea, I updated!